estimagic/optimization/optimize.py

```diff
@@ -177,7 +177,7 @@ def minimize(
         )
         outer_server_process.start()
 
-    result, timing_info = _minimize(
```

Comment:
Does this belong in this PR?

Reply:
Not really. I started the PR to tackle several small issues and then realized that the other issues would need a different reviewer. But I would leave it in. It's really just a small clean-up.

Reply:
Just wanted to make sure it would not have any unwanted side effects. I just did not have anything more substantive to say I guess :-)